data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg

```diff
@@ -42,16 +42,17 @@
         {STORY_PART_TALLIN (_ "This kidnapped princess explains a great deal. It is not easy to draw the elves out of their forests.")}
 
         {STORY_PART_HAMEL (_ "Aye, lad. Her kin will have come either to do battle or pay ransom. By the looks of things, they’ve busked themselves for either.")}
+
```

Comment:
Why is there an extra space here? :o

Reply:
Is that an extra space? Ok I removed it.